tests/test_other.py

```diff
@@ -10233,6 +10233,63 @@ def test_split_main_module(self):
     self.assertIn('Hello from main!', result)
     self.assertIn('Hello from lib!', result)
 
+  def test_split_module_ex(self):
+    initialTableSize = 5632
+    self.set_setting('SPLIT_MODULE')
+    self.set_setting('WASM', 1)
+    self.set_setting('WASM_BIGINT')
+    self.set_setting('EVAL_CTORS', 0)
+    self.set_setting('INLINING_LIMIT', 1)
+    self.set_setting('FORCE_FILESYSTEM', 1)
+    self.set_setting('LEGACY_GL_EMULATION', 0)
+    self.set_setting('DISABLE_EXCEPTION_CATCHING', 0)
+    self.set_setting('ALLOW_MEMORY_GROWTH', 1)
+    self.set_setting('NO_EXIT_RUNTIME', 0)
+    self.set_setting('GL_TRACK_ERRORS', 0)
+    self.set_setting('WASM_OBJECT_FILES', 0)
+    self.set_setting('INCLUDE_FULL_LIBRARY', 1) # https://github.com/emscripten-core/emscripten/issues/13794
+    self.emcc_args += ['-sMAIN_MODULE=2']
+    # Exception: FROZEN_CACHE is set, but cache file is missing: sysroot/lib/wasm32-emscripten/lto-pic/libgl-webgl2-full_es3.a
+    # self.emcc_args += ['-sUSE_WEBGL2']
+    # self.emcc_args += ['-sUSE_GLFW=3']
+    # self.emcc_args += ['-sFULL_ES3=1']
```

Comment:
Do you remember what these were testing?   Were they related to the INCLUDE_FULL_LIBRARY issue?